### ISSUE TITLE
Disable dates

### ISSUE THREAD

**Issue by nohaGazzaz** [balbarak/bootstrap-hijri-datepicker#3]
how to disable certain date ranges

**Comment by balbarak** [comment]
For now you can specifiy the option `minDate` and `maxDate` for example

```js

$("#input")..hijriDatePicker({
    minDate: "2012-01-25"
    maxDate: "2013-02-24"
})

Thanks

**Comment by Bedo-11** [comment]
How can i disable all Friday and Saturday?

**Comment by Shane2021** [comment]
Dears,

Is this project still active as there was no activity on this for long time.

There is an option in the source file of bootstrap-hijri-datetimepicker.js for disabledDates which takes an array but its not working. Any help would be appreciated,

Regards,

**Comment by Shane2021** [comment]
> How can i disable all Friday and Saturday?

There is an option daysOfWeekDisabled which takes range like [0,6] but it seems not working currently. @balbarak may advise.

**Comment by balbarak** [comment]
Yup this project is under development and a new release will be out soon with many fixes and more options

Thanks

**Comment by Shane2021** [comment]
Thanks for your quick reply. Appreciated.

When the new release is expected?

**Comment by Shane2021** [comment]
> Thanks for your quick reply. Appreciated.
> 
> When the new release is expected?

@balbarak
